Makefile

```diff
@@ -52,6 +55,10 @@ test:
 	export PYTHONPATH=./backend:/./tests && \
 	python -m pytest -v -ra tests/
 
+integration-test: install-integration-tests
+	export PYTHONPATH=./backend:/./tests_new && \
+	python -m pytest -v -ra tests_new/integration_tests/
```

Comment:
let's export the XML to make it available in CodeBuild, see [here](https://docs.aws.amazon.com/codebuild/latest/userguide/test-report-pytest.html)

Reply:
Done!